content/en/mTLS (Mutual Transport Layer Security).md

```diff
@@ -0,0 +1,14 @@
+---
+title: mTLS (Mutual Transport Layer Security)
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Mutual TLS (mTLS) is a technique used to authenticate and encrypt messages sent between two [services](https://glossary.cncf.io/service/). Mutual TLS is the standard Transport Layer Security (TLS) protocol but, instead of validating the identity of just one connection, both sides are validated.
+
+## Problem it addresses
+[Microservices](https://glossary.cncf.io/microservices/) communicate over a network and, just like your wifi network, communication in transit over that network can be hacked. mTLS ensures that no unauthorized party can send potentially malicious [API](https://glossary.cncf.io/application_programming_interface/) requests.
```

Comment:
Can we change "mTLS ensures that no unauthorized party can send potentially malicious [API](https://glossary.cncf.io/application_programming_interface/) requests." to "mTLS ensures that no unauthorized party can listen in on or impersonate legitimate requests."

Reply:
Sure :)